includes/polling/applications.inc.php

```diff
@@ -1,13 +1,22 @@
 <?php
 
+use App\Models\Application;
+
 $app_rows = dbFetchRows('SELECT * FROM `applications` WHERE `device_id`  = ?', [$device['device_id']]);
```

Comment:
This query should be 
`$app_rows = \DeviceCache::primary()->applications;`

or more succinctly:
```php
\DeviceCache::primary()->applications->each(function ($app) {
        echo 'Application: ' . $app->app_type. ', app_id=' . $app->app_id;
        $app_include = base_path('includes/polling/applications/' . \LibreNMS\Util\Clean::fileName($app->app_type) . '.inc.php');
        if (is_file($app_include)) {
            include $app_include;
        } else {
            echo 'ERROR: ' . $app_include . ' include file missing!';
        }
});
```

Models implement array access so array style access will work (although it is not preferred)

Reply:
With this change, you never need to call `Application::find($app['app_id'])`

Just access $app->data directly